packages/neuron-wallet/src/services/cells.ts

```diff
@@ -481,7 +481,7 @@ export default class CellsService {
       .where(
         `
         multisig_output.status IN (:...statuses) AND
-        multisig_output.lockArgs in (:lockArgs) AND
+        multisig_output.lockArgs in (:...lockArgs) AND
```

Comment:
This update should be covered by tests

Reply:
As with status, lockargs should be understood as arrays, so after upgrading to a new version, the original usage will not be recognized. So it was corrected. All of the original test case logic can be tested against this change.

Reply:
Got it